testsV2/ut_repyv2api_udp_test_datagram_sizes.r2py

```diff
@@ -0,0 +1,67 @@
+"""
+This tries to send UDP datagrams of varying sizes and establish sane 
+upper limits for a message length check.
+
+On most platforms, UDP over IPv4 can possibly handle: 65,507 bytes =
+  65,535 bytes (per its 16 bit "length" field)
+     - 8 bytes for the UDP header
+     -20 bytes for the IPv4 header.
+
+The limit on OS X is instead 9216, which you can see by executing
+     ` sysctl net.inet.udp.maxdgram `
+on an OS X system. Because we value consistency across platforms and think that
+behavior should be indistinguishable across platforms, we apply this limitation
+to all platforms, so 9217 and above should fail via RepyArgumentError from
+sendmessage.
+
+IPv6 jumbograms allow for UDP datagrams larger than that, but we ignore 
```

Comment:
This bit of information might be good to mention in `emulcomm.py`, possibly as a TODO or limitation of the current implementation.  Developers / reviewers are likely to read the comments in non-test modules, so it's good to get into the habit of having this sort of information in those modules instead.

Reply:
Great point; reproduced in `emulcomm` now.